src/typescript/azure-openapi-validator/rules/DescriptionMustNotBeNodeName.ts

```diff
@@ -3,41 +3,40 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 import { MergeStates, OpenApiTypes, rules } from '../rule';
+import { trimDescription } from './utilities';
+
 export const DescriptionMustNotBeNodeName: string = "DescriptionMustNotBeNodeName";
+
 rules.push({
-  id: "R3011",
+  id: "D4001",
   name: DescriptionMustNotBeNodeName,
   severity: "error",
-  category: "RPCViolation",
+  category: "DocViolation",
   mergeState: MergeStates.composed,
-  openapiType: OpenApiTypes.arm,
+  openapiType: OpenApiTypes.doc,
```

Comment:
does this apply to ARM and data plane specs, any spec? if so, the value here may need to be default or "arm and dataplane"